GenumKit/Parsers/ColorsFileParser.swift

```diff
@@ -66,3 +73,47 @@ public final class ColorsFileParser {
     return value
   }
 }
+
+//MARK: - CLR File Parser
+
+public final class CLRFileParser: ColorsFileParser {
+  public private(set) var colors = [String: UInt32]()
+  
+  public init() {}
+  
+  public func parseFile(path: String) {
+    if let colorsList = NSColorList(name: "UserColors", fromFile: path) {
+      for keyColorPair in colorsList.allKeys.map({ ($0, colorsList.colorWithKey($0)) }) {
+        let colorName = keyColorPair.0.stringByReplacingOccurrencesOfString(" ", withString: "")
```

Comment:
In palettes who knows what users can have. At least we provide some sanity checks. Of course if user have Color#1 like names it will not work anyway.